sdcm/kafka/kafka_cluster.py

```diff
@@ -0,0 +1,171 @@
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation; either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+# Copyright (c) 2023 ScyllaDB
+import logging
+from pathlib import Path
+
+import requests
+
+from sdcm import cluster
+from sdcm.wait import wait_for
+from sdcm.remote import LOCALRUNNER
+from sdcm.utils.git import clone_repo
+from sdcm.utils.common import get_sct_root_path
+from sdcm.utils.remote_logger import DockerComposeLogger
+from sdcm.kafka.kafka_config import SctKafkaConfiguration
+
+# TODO: write/think more about the consumers
+
+LOGGER = logging.getLogger(__name__)
+
+
+class LocalKafkaCluster(cluster.BaseCluster):
+    def __init__(self, remoter=LOCALRUNNER):
+        super().__init__(cluster_prefix="kafka", add_nodes=False)
+        self.remoter = remoter
+        self.docker_compose_path = (
+            Path(get_sct_root_path()) / "kafka-stack-docker-compose"
+        )
+        self._journal_thread: DockerComposeLogger | None = None
+        self.init_repository()
+
+    def init_repository(self):
```

Comment:
why we need to clone whole repository? don't we need only `full-stack.yml` and `conduktor.yml` file?
(btw. how conduktor is used in our context? maybe we could skip it?)

Reply:
The idea is that we can followup on updates

We can later optimize it if we find it an issue (i.e. remove the unneeded files from our fork)

Reply:
it's simple repo, 2 files needed, why we can't follow up manually? This will be similar experience. Just need some docs for that (anyway needed)

Reply:
the repo is small enough, not going to change it.